eng/pipelines/aggregate-report.yml

```diff
@@ -0,0 +1,48 @@
+trigger: none
+pr: none
+
+jobs:
+  - job: Generate
+    variables:
+      - template: templates/variables/globals.yml
+    pool:
+      vmImage: 'windows-2019'
+    steps:
+      - pwsh: mkdir $(System.ArtifactsDirectory)/BuildArtifacts
+        displayName: Create Directory
+
+      - task: DownloadPipelineArtifact@2
+        displayName: 'Download Pipeline Artifact'
+        inputs:
+          buildType: specific
+          project: $(TargetPipelineProjectId)
```

Comment:
Which pipeline is this pulling? Do you know if it only pulls latest successful build?

Reply:
It pulls the latest successful master branch build of the net-core and downloads the artifacts called 'packages'.
I didn't use `allowPartiallySucceededBuilds` or `allowFailedBuilds` so there is not case where it should download an unsuccessful run.